src/step.hpp

```diff
@@ -49,29 +49,29 @@ template<bool O3D> HOST_DEVICE inline void DepthInterfaceCrossing(
         if(ssp->z[iSeg0.z] > DEP(x) && (!O3D || iSeg0.z > 0)) {
             hInt = (ssp->z[iSeg0.z] - DEP(x0)) / DEP(urayt);
 #ifdef STEP_DEBUGGING
-            GlobalLog(
+            printf(
```

Comment:
Where will printf end up when we run in, e.g., Unreal?
There are lots of printf's throughout.

Reply:
There are no `printf`s in the actual compiled version--they are all either preprocessor'd-out in the `STEP_DEBUGGING` system or commented out. Part of the reason we needed a full error/warning handling system was because `printf`s are not accessible within Unreal.